packages/ckeditor5-list/src/listformatting/listitemfontfamilyintegration.ts

```diff
@@ -0,0 +1,77 @@
+/**
+ * @license Copyright (c) 2003-2025, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-licensing-options
+ */
+
+/**
+ * @module list/listformatting/listitemfontfamilyintegration
+ */
+
+import { Plugin } from 'ckeditor5/src/core.js';
+import ListEditing from '../list/listediting.js';
+
+/**
+ * The list item font family integration plugin.
+ */
+export default class ListItemFontFamilyIntegration extends Plugin {
+	/**
+	 * @inheritDoc
+	 */
+	public static get pluginName() {
+		return 'ListItemFontFamilyIntegration' as const;
+	}
+
+	/**
+	 * @inheritDoc
+	 */
+	public static override get isOfficialPlugin(): true {
+		return true;
+	}
+
+	/**
+	 * @inheritDoc
+	 */
+	public static get requires() {
+		return [ ListEditing ] as const;
+	}
+
+	/**
+	 * @inheritDoc
+	 */
+	public init(): void {
+		const listEditing = this.editor.plugins.get( ListEditing );
+
+		// Register the downcast strategy in init() so that the attribute name is registered  before the list editing
+		// registers its converters.
+		// This ensures that the attribute is recognized by downcast strategies and bogus paragraphs are handled correctly.
+		listEditing.registerDowncastStrategy( {
```

Comment:
I guess we could end early and not register the downcast strategy when there is no `FontFamilyEditing` plugin.